src/main/java/com/epam/ta/reportportal/core/project/impl/DeleteProjectHandler.java

```diff
@@ -68,6 +69,7 @@ public OperationCompletionRS deleteProject(String projectName) {
 
 		Project project = projectRepository.findOne(projectName);
 		expect(project, notNull()).verify(PROJECT_NOT_FOUND, projectName);
+		expect(project.getConfiguration().getEntryType(), not(equalTo(EntryType.PERSONAL))).verify(UNABLE_TO_UPDATE_DEFAULT_PROJECT);
```

Comment:
Error message is correct: Unable to update or delete DEFAULT project of Report Portal. {}"